src/main/target/EACHIF3/target.h

```diff
@@ -0,0 +1,133 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define TARGET_BOARD_IDENTIFIER "EACH"  /* https://github.com/vladisenko/EachiWhoop */
+
+#define LED0_PIN                PA8
+
+
+#define USE_EXTI
+#define MPU_INT_EXTI            PA15
+#define USE_MPU_DATA_READY_SIGNAL
+
+
+#define USE_SPI
+#define USE_SPI_DEVICE_1
+#define USE_SPI_DEVICE_2
+
+
+#define MPU6500_SPI_INSTANCE    SPI1
+#define SPI1_SCK_PIN            PB3
+#define SPI1_MISO_PIN           PB4
+#define SPI1_MOSI_PIN           PB5
+#define MPU6500_CS_PIN          PA5
+
+
+#define GYRO
+#define USE_GYRO_SPI_MPU6500
+#define GYRO_MPU6500_ALIGN      CW90_DEG
+
+#define ACC
+#define USE_ACC_SPI_MPU6500
+#define ACC_MPU6500_ALIGN       CW90_DEG
+
+
+#define USE_RX_SPI
+#define USE_RX_A7105
+#define USE_AFHDS2A
+
+#ifdef USE_AFHDS2A
+#define FLYSKY_CHANNEL_COUNT    10
+#define USE_RX_FLYSKY_A2
+#define RX_SPI_DEFAULT_PROTOCOL RX_SPI_A7105_FLYSKY_A2
+#else

```

Comment:
This should not be conditional. Both drivers should be built into the firmware, so that users can choose which protocol to use in the configurator.

Asking end users to recompile the firmware for their craft in order to be able to use FlySky A2 with it is not going to work.